Kernel/Modules/AdminProcessManagementTransitionAction.pm

```diff
@@ -747,4 +770,30 @@ sub _CheckTransitionActionUsage {
     return \@Usage;
 }
 
+sub _GetDefaultConfigParameters {
+    my ( $Self, %Param ) = @_;
+    
+    for my $Needed (qw(Module)) {
+        if ( !$Param{$Needed} ) {
+            $Kernel::OM->Get('Kernel::System::Log')->Log(
+                Priority => 'error',
+                Message  => 'Need '.$Needed.'!'
+            );
+            return;
+        }
+    }
+
+    # replace e.g. 'Kernel::System::ProcessManagement::TransitionAction::TicketCreate' to 'TicketCreate'
+    if ($Param{Module} =~ m/TransitionAction::(.+)$/) {
+        my $Settings = $Kernel::OM->Get('Kernel::Config')->Get('ProcessManagement::TransitionAction::DefaultParameters');
+        return if !IsHashRefWithData($Settings);
+        return if !IsHashRefWithData($Settings->{'001-Framework'});
+        return if !IsHashRefWithData($Settings->{'001-Framework'}->{ $1 });
+            
+        return %{ $Settings->{'001-Framework'}->{ $1 } };
+
+    }
+    return;
```

Comment:
It's a good idea to allow custom sysconfig settings. Instead of creating a new HashReference I would check the config first  `if (IsHashRefWithData($Config->{$Key})) {` and then add it directly to the Setting-Hash `%Settings = ( %Settings, %{ $Config->{$Key} } );`

Reply:
++ 👍🏼  👀 
i like that.